packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -556,13 +556,59 @@ void main() {
 
     await tester.pumpWidget(build(controller));
 
-    expect(tester.getTopLeft(find.text('Alabama')), const Offset(-100.0, 0.0));
-    expect(tester.getBottomRight(find.text('Alabama')), const Offset(900.0, 600.0));
+    expect(tester.getTopLeft(find.text('Alabama')), Offset.zero);
+    expect(tester.getBottomRight(find.text('Alabama')), const Offset(1000.0, 600.0));
 
     controller.jumpToPage(10);
     await tester.pump();
 
-    expect(tester.getTopLeft(find.text('Hawaii')), const Offset(-100.0, 0.0));
+    expect(tester.getTopLeft(find.text('Hawaii')), Offset.zero);
+  });
+
+  testWidgets('All visible pages are able to receive touch events',
+    (WidgetTester tester) async {
```

Comment:
nit: either place this at the end of the previous line (preferred) or indent this more.